content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -1,21 +1,20 @@
 +++
-title = "Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto"
-description = "Instructions for installing Kubeflow with kfctl_existing_arrikto.yaml config"
+title = "Multi-user, auth-enabled Kubeflow with kfctl_istio_dex"
+description = "Instructions for installing Kubeflow with kfctl_istio_dex.yaml config"
 weight = 4
 +++
 
 Follow these instructions if you want to install Kubeflow on an existing Kubernetes cluster.
 
-This installation of Kubeflow is maintained by
-[Arrikto](https://www.arrikto.com/), it is geared towards existing Kubernetes
+This installation of Kubeflow is geared towards existing Kubernetes
 clusters and does not depend on any cloud-specific feature.
```

Comment:
+1 to the comment from @yanniszark that we should help existing users understand that there's a transition from the previous config in v0.7 and the new config in v1.0. I can't find the previous suggested wording. I think we need to add a note at the top of this file, and keep the note as simple as possible. 

Also +1 to the redirect suggested by @jlewi and @krishnadurai in this PR. 

I'm not completely aware of all the complexities, so please excuse me if I inadvertently tread on any toes. Does the following note work, for adding after line 10?

```
**Note:** This installation uses the `kfctl_istio_dex` configuration (configuration file {{% config-uri-istio-dex %}}). This configuration replaces the `kfctl_existing_arrikto` configuration that was used in Kubeflow v0.7 and earlier.
```

Reply:
@sarahmaddox Wouldn't it be simpler to just mention that kfctl_existing_arrikto is no longer supported?

Reply:
@krishnadurai I think it's clearer to tell people that this config replaces the earlier one. The current wording seems out of place, because it's talking about a config that's not mentioned anywhere else. People will wonder why we're mentioning it:
`Note: The configuration kfctl_existing_arrikto that was used in Kubeflow v0.7 and earlier is no longer supported.`

The wording in my earlier comment mentions that the new config is a replacement for the earlier one.

In a later comment, @yanniszark has said he's OK with my suggested wording too. I recommend we go with it.